pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -115,8 +123,9 @@ public SuccessResponse deleteSchema(
   @ApiResponses(value = {@ApiResponse(code = 200, message = "Successfully updated schema"), @ApiResponse(code = 404, message = "Schema not found"), @ApiResponse(code = 400, message = "Missing or invalid request body"), @ApiResponse(code = 500, message = "Internal error")})
   public SuccessResponse updateSchema(
       @ApiParam(value = "Name of the schema", required = true) @PathParam("schemaName") String schemaName,
-      FormDataMultiPart multiPart) {
-    return addOrUpdateSchema(schemaName, multiPart);
+      FormDataMultiPart multiPart,
+      @ApiParam(value = "Whether to force to reload segments") @DefaultValue("false") @QueryParam("forceReload") Boolean forceReload) {
```

Comment:
Can we make this parameter **boolean** instead **Boolean**? Also I would recommend name it **"reload"**.